diff --git a/chrome/browser/content/browser/browser.xhtml b/chrome/browser/content/browser/browser.xhtml
index 5128ff5..3ac9f36 100644
--- a/chrome/browser/content/browser/browser.xhtml
+++ b/chrome/browser/content/browser/browser.xhtml
@@ -294,10 +294,12 @@ if (AppConstants.platform == "macosx") {
          modifiers="accel" reserved="true"/>
     <key id="key_newNavigatorTab" data-l10n-id="tab-new-shortcut" modifiers="accel"
          command="cmd_newNavigatorTabNoEvent" reserved="true"/>
+<!--
     <key id="focusURLBar" data-l10n-id="location-open-shortcut" command="Browser:OpenLocation"
          modifiers="accel"/>
     <key id="focusURLBar2" data-l10n-id="location-open-shortcut-alt" command="Browser:OpenLocation"
          modifiers="alt"/>
+-->
 
     <key id="key_search" data-l10n-id="search-focus-shortcut" command="Tools:Search" modifiers="accel"/>
     <key id="key_search2"
@@ -419,6 +421,7 @@ if (AppConstants.platform == "macosx") {
     <key id="key_undoCloseWindow" command="History:UndoCloseWindow" data-l10n-id="window-new-shortcut" modifiers="accel,shift"/>
 
 
+<!--
 <key id="key_selectTab1" oncommand="gBrowser.selectTabAtIndex(0, event);" key="1" modifiers="alt"/>
 <key id="key_selectTab2" oncommand="gBrowser.selectTabAtIndex(1, event);" key="2" modifiers="alt"/>
 <key id="key_selectTab3" oncommand="gBrowser.selectTabAtIndex(2, event);" key="3" modifiers="alt"/>
@@ -428,6 +431,7 @@ if (AppConstants.platform == "macosx") {
 <key id="key_selectTab7" oncommand="gBrowser.selectTabAtIndex(6, event);" key="7" modifiers="alt"/>
 <key id="key_selectTab8" oncommand="gBrowser.selectTabAtIndex(7, event);" key="8" modifiers="alt"/>
 <key id="key_selectLastTab" oncommand="gBrowser.selectTabAtIndex(-1, event);" key="9" modifiers="alt"/>
+-->
 
     <key id="key_wrCaptureCmd"
     key="#" modifiers="control"
